spec/features/audit/report/progress_spec.rb

```diff
@@ -1,72 +1,102 @@
 RSpec.feature "Reporting on audit progress" do
-  let!(:my_organisation) do
-    create(
-      :organisation
-    )
+  scenario "Displaying the number of items included in the audit" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    and_one_content_item_needs_improvement_and_one_does_not
+    when_i_visit_the_report_page
+    then_i_see_the_report_includes_all_three_content_items
+    when_i_select_a_document_type_that_does_not_apply_to_any_content_item
+    then_i_see_the_report_includes_no_content_items
+    when_i_select_a_document_type_that_applies_to_all_three_content_items
+    then_i_see_the_report_includes_all_three_content_items
   end
 
-  let!(:me) do
-    create(
-      :user,
-      organisation: my_organisation,
-    )
+  scenario "Displaying the number of items audited/not audited" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    and_two_items_have_been_audited
+    when_i_visit_the_report_page
+    then_i_see_that_67_percent_of_my_audit_is_completed
+    and_i_see_that_33_percent_of_my_audit_is_not_completed
   end
 
-  context "I have been allocated some audited and unaudited content items belonging to my organisation" do
-    before(:each) do
-      content_items = create_list(
-        :content_item,
-        3,
-        document_type: "transaction",
-        allocated_to: me,
-        primary_publishing_organisation: my_organisation,
-      )
+  scenario "Displaying the number of items needing improvement/not needing improvement" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    and_one_content_item_needs_improvement_and_one_does_not
+    when_i_visit_the_report_page
+    then_i_see_that_50_percent_of_content_needs_improvement
+    and_i_see_that_50_percent_of_content_does_not_need_improvement
+  end
 
-      create(:passing_audit, content_item: content_items[0])
-      create(:failing_audit, content_item: content_items[1])
-    end
+  scenario "Audit status filter is not present" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    when_i_visit_the_report_page
+    then_i_do_not_see_audit_status_filter
+  end
 
-    scenario "Displaying the number of items included in the audit" do
-      visit audits_report_path
+  def then_i_see_the_report_includes_all_three_content_items
+    expect(@audit_report_page).to have_report_section(text: '3 Content items')
+  end
 
-      expect(page).to have_content("3 Content items")
+  def then_i_see_the_report_includes_no_content_items
+    expect(@audit_report_page).to have_report_section(text: '0 Content items')
```

Comment:
`expect(@audit_report_page).to have_content_item_count(0)` ?

Reply:
√